File: app/src/main/java/com/dsm2018/get_terra_android_v2/AccurateActivity.java
package com.dsm2018.get_terra_android_v2;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;


public class AccurateActivity extends Activity {
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_accurate);

        Button btn_gomian = (Button)findViewById(R.id.btn_main);
        btn_gomian.setOnClickListener(new Button.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(AccurateActivity.this, MainActivity.class);
                startActivity(intent);
            }
        });

    }
}

File: app/src/main/java/com/dsm2018/get_terra_android_v2/AccurateActivity.kt
package com.dsm2018.get_terra_android_v2

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button


class AccurateActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_accurate)

        val btn_gomian = findViewById<View>(R.id.btn_main) as Button
        btn_gomian.setOnClickListener {
            var intent = Intent(this@AccurateActivity, MainActivity::class.java)
        }
        /*btn_gomian.setOnClickListener(object : Button.OnClickListener {
            override fun onClick(view: View) {
                val intent = Intent(this@AccurateActivity, MainActivity::class.java)
                startActivity(intent)
            }
        })*/

    }
}

